connect/src/test/java/com/stripe/android/connect/webview/StripeConnectWebViewContainerControllerTest.kt

```diff
@@ -58,6 +66,10 @@ class StripeConnectWebViewContainerControllerTest {
 
     @Before
     fun setup() {
+        Dispatchers.setMain(Dispatchers.Unconfined)
```

Comment:
Shouldn't be necessary if we remove setting the dispatcher as suggested in my other comment. Otherwise, we'll need to `Dispatchers.resetMain()` in an `@After` block.

Reply:
This is still necessary - see comment above. I've added an `@After` as well now